sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "What's new in Svelte: January 2022"
+description: "SvelteKit 1.0, learn.svelte.dev, and type definitions for Svelte elements."
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+It's been just two weeks since the release of [SvelteKit 1.0](https://svelte.dev/blog/announcing-sveltekit-1.0)! If you haven't yet, check out the [livestream](https://www.youtube.com/watch?v=N4BRVkQVoMc), [new website](https://kit.svelte.dev/) and [learn.svelte.dev](https://learn.svelte.dev/) to learn all the features of SvelteKit step-by-step.
+
+Or, if you prefer, just [subscribe to this newsletter](https://svelte.substack.com/) 😉 Let's dive into the details...
```

Comment:
I'm not sure this sentence really meshes with the rest. Subscribing will give you future updates, but it won't tell you about the stuff that happened this month, which you can just read below. I think if we were going to have a link we'd probably want to make it be https://svelte.dev/blog/rss.xml as the people who have chosen to read on svelte.dev might prefer to read it there in the future as well.
```suggestion
Let's dive into the details...
```

Reply:
Good call. Removing